src/modules/70-pipeline/components/ArtifactsSelection/ArtifactUtils.ts

```diff
@@ -107,6 +107,15 @@ export const shouldFetchTags = (
   )
 }
 
+export const shouldFetchTagsSource = (connectorRefValue: any, queryParamList: Array<string>): boolean => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9254512)
```suggestion
export const shouldFetchTagsSource = (connectorRefValue: unknown, queryParamList: Array<string>): boolean => {
```